image/png.ksy

```diff
@@ -292,6 +297,56 @@ types:
       - id: text_datastream
         process: zlib
         size-eos: true
+  animation_control_chunk:
+    doc-ref: https://wiki.mozilla.org/APNG_Specification#.60acTL.60:_The_Animation_Control_Chunk
+    seq:
+      - id: num_frames
+        type: u4
+        doc: Number of frames
+      - id: num_plays
+        type: u4
+        doc: Number of times to loop, 0 indicates infinite looping.
+  frame_control_chunk:
+    doc-ref: https://wiki.mozilla.org/APNG_Specification#.60fcTL.60:_The_Frame_Control_Chunk
+    seq:
+      - id: sequence_number
+        type: u4
+        doc: Sequence number of the animation chunk
+      - id: width
+        type: u4
+        doc: Width of the following frame
+      - id: height
+        type: u4
+        doc: Height of the following frame
+      - id: x_offset
+        type: u4
+        doc: X position at which to render the following frame
+      - id: y_offset
+        type: u4
+        doc: Y position at which to render the following frame
+      - id: delay_num
+        type: u2
+        doc: Frame delay fraction numerator
+      - id: delay_den
+        type: u2
+        doc: Frame delay fraction denominator
```

Comment:
It would be great to clarify which time units does frame delay use in this case.

Reply:
I will try to clarify this. The specification says:

"he `delay_num` and `delay_den` parameters together specify a fraction indicating the time to display the current frame, in seconds."

Reply:
I guess it may make sense to introduce the type `fraction`

Reply:
> I think this is probably best documented in the value instance as discussed below.

The value instance helps, but @GreyCat probably meant that you could also write `doc: in seconds` to the value instance so that it is clear in what units the final decimal value is measured.